package.json

```diff
@@ -64,6 +73,7 @@
     "nyc": "^11.3.0",
     "portfinder": "^1.0.13",
     "pre-commit": "^1.2.2",
+    "proxyquire": "^1.8.0",
```

Comment:
duplicate dependency functionality with mock-require
please remove proxyquire and refactor your tests to use mock-require